pkgs/development/libraries/gspell/default.nix

```diff
@@ -23,18 +26,24 @@ stdenv.mkDerivation rec {
     sha256 = "1pdb4gbjrs8mk6r0ipw5vxyvzav1wvkjq46kiq53r3nyznfpdfyw";
   };
 
+  patches = [
+    ./0001-Darwin-build-fix.patch
```

Comment:
Could you please try submitting the patch upstream?

Reply:
https://gitlab.gnome.org/GNOME/gspell/-/blob/30f66b828173c6ea7c1f413503d80e8fb02cc74e/HACKING

```
gspell development
==================

Contributions are no longer accepted
------------------------------------

And new maintainers are not accepted either.
```

<details>
  <summary>I was also trying to get the latest gspell commit to build so I could make sure the patch works there too before I submit it, but it just spits autoconf errors at me that I can't solve.</summary>

```
autoreconf: running: /nix/store/1hbdwchm6mkqhn8ygdgjjj9kkmps8ilv-autoconf-2.71/bin/autoconf --force
configure.ac:58: warning: The macro `AC_CONFIG_HEADER' is obsolete.
configure.ac:58: You should run autoupdate.
./lib/autoconf/status.m4:719: AC_CONFIG_HEADER is expanded from...
configure.ac:58: the top level
configure.ac:126: warning: AX_ADD_AM_MACRO_STATIC is m4_require'd but not m4_defun'd
m4/ax_code_coverage.m4:208: _AX_CODE_COVERAGE_ENABLED is expanded from...
lib/m4sugar/m4sh.m4:699: AS_IF is expanded from...
m4/ax_code_coverage.m4:248: AX_CODE_COVERAGE is expanded from...
configure.ac:126: the top level
configure.ac:150: warning: The macro `AC_TRY_COMPILE' is obsolete.
configure.ac:150: You should run autoupdate.
./lib/autoconf/general.m4:2847: AC_TRY_COMPILE is expanded from...
configure.ac:150: the top level
configure.ac:70: error: possibly undefined macro: PKG_PROG_PKG_CONFIG
      If this token and others are legitimate, please use m4_pattern_allow.
      See the Autoconf documentation.
configure.ac:71: error: possibly undefined macro: PKG_INSTALLDIR
configure:16724: error: possibly undefined macro: AC_MSG_ERROR
configure:17035: error: possibly undefined macro: AM_V_at
configure:17053: error: possibly undefined macro: AM_DISTCHECK_CONFIGURE_FLAGS
autoreconf: error: /nix/store/1hbdwchm6mkqhn8ygdgjjj9kkmps8ilv-autoconf-2.71/bin/autoconf failed with exit status: 1
```

</details>

Reply:
The original commit i looked at that fixed gspell (https://github.com/mjlbach/nixpkgs/commit/5cf11fc91af7effed5cb2ccb3d4cb9ffa0e100b3) didn't specify where the patch is from but it [seems to be from Homebrew, embedded in their build script](https://github.com/Homebrew/homebrew-core/blob/2a27fb86b08afc7ae6dff79cf64aafb8ecc93275/Formula/gspell.rb#L125-L149). I can try to fetchpatch their recipe & extract the patch in postFetch instead if you'd prefer that?

Reply:
At least add a link from where you got the patch incase it ever needs an update.

Reply:
Done.